namer/core/src/main/scala/io/buoyant/namer/ConfiguredDtabNamer.scala

```diff
@@ -2,13 +2,17 @@ package io.buoyant.namer
 
 import com.twitter.finagle.Name.Bound
 import com.twitter.finagle._
+import com.twitter.finagle.http.{Request, Response, Status}
 import com.twitter.finagle.naming.NameInterpreter
 import com.twitter.util.{Activity, Future, Var}
+import io.buoyant.admin.Admin
 
 case class ConfiguredDtabNamer(
   configuredDtab: Activity[Dtab],
-  namers: Seq[(Path, Namer)] = Nil
-) extends NameInterpreter with Delegator {
+  namers: Seq[(Path, Namer)] = Nil,
+  uri: Option[String] = None,
```

Comment:
instead of passing in an Option[String] and Option[Service], what about just passing in a Seq[Admin.Handler]?

Reply:
That is much cleaner actually, Thanks!